inspirehep/modules/theme/static/scss/format/brief-results.scss

```diff
@@ -116,27 +114,29 @@ hr {
     cursor:pointer;
 }
 
-@media only screen and (max-width : 992px){
-    .right-column {
-        display:none;
+/* Mobile styles
+-------------------------------------------------- */
+@media only screen and (max-width : $screen-md-min) {
+
+    #header-section {
+        padding-bottom: 50px;
     }
 
-    .abstract {
-        display:none;
+    #search-results #results-control-panel .panel-body .row > div {
+        text-align: center;
+        margin-bottom: 10px;   
     }
+}
 
-    .title {
-        display:none;
-    }   
+@media (max-width: 600px) {
```

Comment:
In this case no, because under <600px the search bar is shifted to the second 'line':

![capture d ecran 2016-03-17 a 14 23 33](https://cloud.githubusercontent.com/assets/2742143/13847168/1250ad84-ec4c-11e5-9a32-628ef8f28fc0.png)